blog/tutorial/01-index.md

```diff
@@ -450,17 +462,17 @@ Note: depending on your OS and firewall settings, you may have to accept a promp
 Run the following command to create a post:
 
 ```sh
-blogd tx blog create-post "My first post" "This is a post\!" --from=user1
+blogd tx blog create-post "My first post" "This is a post\!" --from=alice
 ```
 
-“This is a post!” is a title for our post and `--from=user1` tells the program who is creating this post. `user1` is a label for your pair of keys used to sign the transaction, created by the initialization script located within the `/Makefile` previously. Keys are stored in `~/.blogd`.
+“My first post” is a title for our post and `--from=alice` tells the program who is creating this post. `alice` is a label for your pair of keys used to sign the transaction, created by the initialization script located within the `/Makefile` previously. Keys are stored in `~/.blogd`.
```

Comment:
```suggestion
“- My first post” is a title for our post
- `--from=alice` is the author of the post
- `alice` is a label for your pair of keys that is used to sign the transaction
- Key pairs were created earlier by the initialization script in the `/Makefile` 
- Keys are stored in `~/.blogd`
```

Reply:
I wasn't clear on what was created earlier... the key pairs or the transaction or ?? make sure I don't introduce errors here, teach me!

> Key pairs were created earlier by the initialization script in the `/Makefile`